stm32-modules/thermocycler-gen2/src/plate_control.cpp

```diff
@@ -98,20 +127,32 @@ auto PlateControl::set_new_target(double setpoint, double hold_time,
     return temp * IDLE_FAN_POWER_SLOPE;
 }
 
+[[nodiscard]] auto PlateControl::calculate_overshoot(double setpoint,
```

Comment:
these can be `const` and maybe even `static` since it looks like they just use static member values and arguments